src/hotspot/os/linux/os_linux.cpp

```diff
@@ -378,22 +378,40 @@ static void next_line(FILE *f) {
   } while (c != '\n' && c != EOF);
 }
 
-void os::Linux::kernel_version(long* major, long* minor) {
+void os::Linux::kernel_version(long* major, long* patch, long* sub) {
```

Comment:
I think semver calls these "major", "minor", "patch".

Reply:
Yes, it seems like it's only the actual define for these numbers which have those names.
I'll change as the kselftest also use the names you suggest.